app/js/ui.js

```diff
@@ -1,29 +1,64 @@
 import 'whatwg-fetch';
 import React from 'react';
+import { autobind } from 'core-decorators';
+import classNames from 'classnames';
+import { App } from 'spak';
 
-class App extends React.Component {
+class AppVC extends React.Component {
     constructor() {
         super();
         this.state = {
-            styles: ''
+            styles: '',
+            hasLoginForm: false
```

Comment:
What are your preferences on trailing commas in a place like this?
Meaning this diff would be one line - the line that's actually changing?